src/lib/metricsMgr/MetricsManager.cpp

```diff
@@ -131,15 +134,54 @@ void MetricsManager::add(const std::string& srv, const std::string& subServ, con
     return;
   }
 
+  // FIXME P4: See github issue #2781
```

Comment:
Probably an elaboration of this beyond the issue number itself could be useful. I mean:

"FIXME P4: we check service and subsrv are legal names, otherwise metrics are not accounted. Doing this each time add() is called could have performance issues, #2781 is about better solutions for this"

or something similar.